src/container/srv_target.c

```diff
@@ -425,6 +429,9 @@ cont_child_aggregate(struct ds_cont_child *cont, cont_aggregate_cb_t agg_cb,
 		DP_CONT(cont->sc_pool->spc_uuid, cont->sc_uuid),
 		tgt_id, epoch_range.epr_lo, epoch_range.epr_hi);
 
+	if (!param->ap_vos_agg)
+		vos_cont_set_mod_bound(cont->sc_hdl, epoch_range.epr_hi);
```

Comment:
Why do we need to set this value here and there? Isn't it a global constant value 'cur_time - gap'? I don't see why it's related to aggregation range.

Reply:
If aggregation run slowly, much behind of current HLC, it is unnecessary to restart the DTX which epoch is out of 'cur_time - gap' but newer than current aggregation boundary. We only need to guarantee that the DTX's epoch is not smaller than current aggregation real boundary.